v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
-    exit 0
+etcdctl get /capcom/config/proxy
```

Comment:
We could just create an entirely new `/flags` top-level dir: `/flags/capcom-bootstrapped` etc